src/NzbDrone.Core.Test/IndexerSearchTests/SearchDefinitionFixture.cs

```diff
@@ -8,16 +9,16 @@ namespace NzbDrone.Core.Test.IndexerSearchTests
 {
     public class SearchDefinitionFixture : CoreTest<SingleEpisodeSearchCriteria>
     {
-        [TestCase("Betty White's Off Their Rockers", Result = "Betty+Whites+Off+Their+Rockers")]
-        [TestCase("Star Wars: The Clone Wars", Result = "Star+Wars+The+Clone+Wars")]
-        [TestCase("Hawaii Five-0", Result = "Hawaii+Five+0")]
-        [TestCase("Franklin & Bash", Result = "Franklin+and+Bash")]
-        [TestCase("Chicago P.D.", Result = "Chicago+PD")]
-        [TestCase("Kourtney And Khlo\u00E9 Take The Hamptons", Result = "Kourtney+And+Khloe+Take+The+Hamptons")]
-        public string should_replace_some_special_characters(string input)
+        [TestCase("Betty White's Off Their Rockers", "Betty+Whites+Off+Their+Rockers")]
+        [TestCase("Star Wars: The Clone Wars", "Star+Wars+The+Clone+Wars")]
+        [TestCase("Hawaii Five-0", "Hawaii+Five+0")]
+        [TestCase("Franklin & Bash", "Franklin+and+Bash")]
+        [TestCase("Chicago P.D.", "Chicago+PD")]
+        [TestCase("Kourtney And Khlo\u00E9 Take The Hamptons", "Kourtney+And+Khloe+Take+The+Hamptons")]
+        public void should_replace_some_special_characters(string input, string expected)
```

Comment:
The old way wasn't supported anymore that I could tell, plus this is more consistent with everything else we do.